static/usr/bin/nvidia-create-device

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh -eux
```

Comment:
Why cannot we just ship ub-create-device?

Reply:
The problem with it is that it depends on the nvidia driver version, so you get different version for 535, 550, etc. The real solution would be to let the kernel add udev rules to the system, but we are not there yet.

Reply:
It is this right? https://github.com/canonical/nvidia-graphics-drivers/blob/560/debian/device-create/ub-device-create.c

That file has not changed for 4 years.

Reply:
Thanks, I actually thought this was coming from nvidia. I've changed this now.